<launch>
  <arg name="robot_name" />
  <arg name="sim" default="false" />
  <arg name="standalone" default="true" />
  
  <group if="$(arg sim)" ns="pan_motor">
    <node name="pan_motor_sim" pkg="basic_sim" type="sim_node" output="screen">
      <param name="load_urdf" value="$(eval not standalone)" />
      <rosparam command="load" file="$(find pan_tilt)/config/sim_pan_driver.yaml" />
    </node>
    
    <rosparam command="load" file="$(find pan_tilt)/config/pan_controllers.yaml" />
    <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false"
          output="screen" args="joint_state_controller position_controller">
    </node>
  </group>

</launch>
